### ISSUE TITLE
Using HaxeUI in OpenFL neko/cpp target forces legacy implementation

### ISSUE THREAD

**Issue by er453r** [ianharrigan/haxeui#299]
I've added HaxeUI to my OpenFL project, and the html5 traget works fine.

The neko/cpp targets seem to force openfl into using legacy implementations of objects and I start to get tons of errors like:

    Class<openfl._legacy.display.BitmapData> has no field fromBytes
    openfl._legacy.display.Stage has no field window

**Comment by ianharrigan** [comment]
Yeah, unfortunately haxeui version 1 forces next legacy at the moment, its worth noting that version 2 doesnt have this limitation.
